scripts/generate_proto.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+# Install protoc
+brew install protobuf
```

Comment:
I wonder if we can encode the telemetry data without the need to import the entire protobuf library

Reply:
Not a big issue if the resulting pb file is checked into the source code (i.e. no extra build steps required for users to compile)